### PR TITLE
Fix file path parsing bug

### LINKED ISSUE #246: [PE-D][Tester C] Import function parsing error
Keying in import with / as its argument throws an error in the terminal but not handled by the UI. This can be shown as the UI does not turn the command red to show that there was an error in the parsing of the command when instead throws a null pointer exception instead. Strangely, of all the characters tested so far, only '/' has the unhandled error

-------------
Labels: `severity.Low` `type.FeatureFlaw`
original: jylow/ped#5

### LINKED ISSUE #247: [PE-D][Tester C] Parsing error with export function
Similar to import function, argument of / is not handled by the parser properly and thus results in null pointer expection that is not being handled properly

-------------
Labels: `severity.Low` `type.FeatureFlaw`
original: jylow/ped#6

### PR DESCRIPTION
**Pull request by aexolate**
Fixes #246 
Fixes #247

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/ExportCommand.java [hunk 1]
```diff
@@ -16,7 +17,10 @@ public class ExportCommand extends Command {
     public static final String COMMAND_WORD = "export";
     public static final String MESSAGE_SUCCESS = "Data successfully exported";
     public static final String MESSAGE_FAILURE = "Data could not be exported";
-    public static final String MESSAGE_INVALID_FILE_TYPE = "File should be of .json type. E.g. data.json";
+    //public static final String MESSAGE_INVALID_FILE_TYPE = "File should be of .json type. E.g. data.json";
```
Comment (Weiennn) [inline src/main/java/seedu/address/logic/commands/ExportCommand.java:20]: Should this be removed?

**Review by Weiennn — APPROVED** [review]
LGTM

**Comment by Goh-Li-Ting** [comment]
I tried running the command `export /test_folder/test_file.json` and the app said that the export was successful but I did not see any files created.

**Comment by aexolate** [comment]
> I tried running the command `export /test_folder/test_file.json` and the app said that the export was successful but I did not see any files created.

Is the file found under C:/test_folder/test_file.json? Assuming you are on windows machine, the '/' at the front means the root folder. I am probably going to add some explanation in the UG

**Comment by Goh-Li-Ting** [comment]
> @Goh-Li-Ting Is the file found under C:/test_folder/test_file.json? Assuming you are on windows machine, the '/' at the front means the root folder. I am probably going to add some explanation in the UG

@aexolate Yup it is, thanks!

**Review by Goh-Li-Ting — APPROVED** [review]
LGTM

**Pull request merged**